charts/kube-state-metrics/values.yaml

```diff
@@ -162,7 +162,7 @@ collectors:
   - statefulsets
   - storageclasses
   - validatingwebhookconfigurations
-  - verticalpodautoscalers
+  # - verticalpodautoscalers # not a default resource
```

Comment:
I would suggest just removing it

Reply:
https://github.com/kubernetes/kube-state-metrics#enabling-verticalpodautoscalers This is a useful link to add to the chart

Reply:
I added your link to the comment and moved it to the end of the values. I would not remove it since others might wonder where it is.